Update Readme to add WSL compatibility disclaimer (#54)
PlayFab/PlayFabParty: README.md

* Android - `/android/bin/release/`
* Win32 - The win32 binaries are not included in the zip assets. They are available on download on [nuget](https://www.nuget.org/packages/Microsoft.PlayFab.PlayFabParty.Cpp.Windows).
* Linux - `/x64/Release/libparty.so`

## Troubleshooting

> ### WSL Compatibility

Reviewer: I would place the "WSL Compatibility" section outside (above) the Troubleshooting section (and remove the troubleshooting section for now), as this is technically not a troubleshooting-related note, but rather a description of the product.

> ### WSL Compatibility

Reviewer 2: Shouldn't these sorts of docs be published to the public MSDN docs instead of this README? it's probably fine to have them in both places but the MSDN docs are the more comprehensive source of truth for API considerations like this.

Author: They should but we haven't allocated the work for that yet. It is logged. This is just a quick fix to support the customers.

> While WSL can be used as a build platform to create Party binaries, it is not supported as a runtime platform to use Party SDK and its features. A machine with a Linux OS is required to take advantage of all Party features.

Reviewer 2: ```suggestion
While WSL can be used to build projects which depend on PlayFab Party binaries, it is not supported as a runtime platform to use the Party SDK. A machine with a Linux OS is required to take advantage of all Party features.
```
since we don't support anyone actually creating their own Party binaries

Author: I'll just remove that first part of the sentence entirely.
